### DISCUSSION TITLE
An error occurred when connecting to the Pockethost admin site.

### DISCUSSION THREAD

**Discussion by Sungwoo00** [pocketbase/pocketbase#6182] [Q&A]
> This article is written through a translator. Sentences may not be smooth.

When I access the Pocket Host admin site, the error below occurs.
`Error: Log in at https://pockethost.io to verify your account.`
How do I solve this?

**Comment by ganigeorgiev** [comment]
It is not clear what actually you mean, but in any case this is not the place for questions about Pockethost (or any other hosting service).

If your issue occur when you are authenticated as admin via the JS SDK, then please see https://github.com/pocketbase/pocketbase/issues/6180, https://github.com/pocketbase/pocketbase/discussions/6110 and probably others. Or in other words, make sure that the used JS SDK and PocketBase executable versions are compatible.

If that doesn't help elaborate what PocketBase executable and JS SDK versions you are using and provide a code sample, but again PocketHost has no relation to the development of PocketBase and I cannot assist you with it.
